quickstart-node/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "monday-code-quickstart",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "tsc",
+    "start": "node -r ts-node/register/transpile-only -r tsconfig-paths/register dist/index.js",
+    "dev": "npm run stop && concurrently \"npm run server\" \"npm run expose\"",
+    "server": "nodemon",
+    "expose": "PORT=$(grep PORT .env | cut -d '=' -f2) && mapps tunnel:create -p ${PORT:-3000}",
+    "stop": "PORT=$(grep PORT .env | cut -d '=' -f2) && kill-port ${PORT:-3000}",
+    "fetch:schema": "bash fetch-schema.sh",
+    "codegen": "graphql-codegen",
+    "fetch:generate": "npm run fetch:schema && npm run codegen"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "nodemonConfig": {
+    "ignore": [
+      "**/*.test.ts",
+      "**/*.spec.ts",
+      ".git",
+      "node_modules"
+    ],
+    "watch": [
+      "src"
+    ],
+    "exec": "node -r tsconfig-paths/register -r ts-node/register ./src/index.ts",
+    "ext": "ts, js"
+  },
+  "dependencies": {
+    "@mondaycom/apps-sdk": "^3.0.11",
+    "@mondaydotcomorg/api": "^7.0.1",
+    "axios": "^1.7.7",
+    "body-parser": "^2.0.1",
+    "cookie-parser": "^1.4.7",
+    "dotenv": "^16.4.5",
```

Comment:
move to dev dependencies

Reply:
It is not a dev dependency, you use it in `index.ts` to load all your envs. I know that in monday code it is working without it because the envs are already set up, but the app will crash because it will try to find the `require("dotenv")` but it is not installed